source/includes/APIReference/TimeOffRequests/Examples/_create-response.md.erb

```diff
@@ -0,0 +1,92 @@
+{
+  "results": {
+    "time_off_requests": {
+      "1546922": {
+        "_status_code": 200,
+        "_status_message": "Created",
+        "id": 1546922,
+        "user_id": 423,
+        "time_off_request_notes": [96024],
+        "time_off_request_entries": [11374, 11375],
+        "status": "pending",
+        "active": true,
+        "created": "2018-11-11T10:56:15-06:00",
+        "last_modified": "2018-11-11T10:56:15-06:00"
+      }
+    }
+  },
+  “supplemental_data”: {
```

Comment:
Wrong quote char sneaked in here somehow

![image](https://user-images.githubusercontent.com/44206128/81846162-e575ba00-950e-11ea-9530-b94ad140f5be.png)